bool data
---
geometry_msgs/PointStamped msg